blocks/highlight-card/highlight-card.js

```diff
@@ -0,0 +1,30 @@
+export default function decorate(block) {
+  block.querySelectorAll('a').forEach((buttonLink) => {
+    buttonLink.classList.add('button');
+    buttonLink.classList.add('secondary');
+    buttonLink.closest('div').classList.add('button-container');
+  });
+
+  let childRowNumber = 0;
+  [...block.children].forEach((row) => {
```

Comment:
the second argument of `forEach` is `index` so you shouldn't have to keep track of `childRowNumber`.